pkg/apis/autoscaling/v2beta2/zz_generated.conversion.go

```diff
@@ -515,7 +611,17 @@ func autoConvert_v2beta2_HorizontalPodAutoscalerSpec_To_autoscaling_HorizontalPo
 	}
 	out.MinReplicas = (*int32)(unsafe.Pointer(in.MinReplicas))
 	out.MaxReplicas = in.MaxReplicas
-	out.Metrics = *(*[]autoscaling.MetricSpec)(unsafe.Pointer(&in.Metrics))
```

Comment:
not sure why this changed... see if this goes back to a pointer cast after removing the `PodResource *PodResourceMetricSource`